recipes/pyzbar/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyzbar" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pyzbar"
+  git_rev: "v{{ version }}"
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
This recipe looks like it could be noarch. Don't worry about skipping windows for noarch. Noarch recipes build on linux only and they won't install unless they have their deps. So as long as the deps do not exist on windows, it won't install. OTOH if the package does not work on windows, then keep it as is.

Reply:
noarch option throws error:
[noarch_python] Noarch package contains binary script: read_zbar.exe